joy_topic: "joy"

# --- Params for brake controller --- #
brake_topic: "brake_controller/data_control"
pos_min: 40
pos_max: 3000

# --- params for dir controller --- #
dir_topic: "control_dir/dir_data"

# --- params for accel puma --- #
accel_puma_topic: "accel_puma/value"
min_accel: 0
max_accel: 100 # max value 126


# --- Params for controller --- #
lt_left_index: 2
rt_right_index: 5

x_left_index: 0
y_left_index: 1

x_right_index: 3
y_right_index: 4

a_button_index: 0
b_button_index: 1
x_button_index: 2
y_button_index: 3
lb_button_index: 4
rb_button_index: 5